### PR TITLE
fix(chat): Update new messages marker when message was removed

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #9014

### What does the PR do

Update `NewMessagesMaker` when some message was deleted

### Affected areas

Chat

### CODE REVIEW

**Review by jrainville — APPROVED** [review]

**Review by osmaczko — APPROVED** [review]
Thank you. Looks good. I had that in my mind and forgot to address it. Could you please remove "# TODO: handle messages removal" from that file?

File: src/app/modules/shared_models/message_model.nim [hunk 1]
```diff
@@ -77,6 +77,8 @@ QtObject:
       [{i}]:({$self.items[i]})
       """
 
+  proc resetNewMessagesMarker*(self: Model)
+
```
Comment (MishkaRogachev) [inline src/app/modules/shared_models/message_model.nim:81]: Isn't it a  `{.signal.}` ?

Comment (osmaczko) [reply src/app/modules/shared_models/message_model.nim:81]: Nope

**Review by MishkaRogachev — APPROVED** [review]

**Comment by borismelnik** [comment]
> Thank you. Looks good. I had that in my mind and forgot to address it. Could you please remove "# TODO: handle messages removal" from that file?

done :)

**Pull request merged**